kitsune/wiki/views.py

```diff
@@ -213,6 +219,7 @@ def document(request, document_slug, template=None, document=None):
         'ga_push': ga_push,
         'breadcrumb_items': breadcrumbs,
         'document_css_class': document_css_class,
+        'fx_download_button': fx_download_button,
```

Comment:
`fx_download_button` is a noun. I would expect this variable to hold a button object. Instead it holds a boolean. A name like `show_fx_download` would be more readable, mostly because it is a verb phrase.